pkg/porter/credentials_test.go

```diff
@@ -231,3 +232,57 @@ good-creds   now`},
 		})
 	}
 }
+
+func TestGenerateNoCredentialDirectory(t *testing.T) {
+	p := NewTestPorter(t)
+	p.TestConfig.SetupPorterHome()
+	p.CNAB = &TestCNABProvider{}
+
+	opts := CredentialOptions{
+		Silent: true,
+	}
+	opts.Name = "name"
+	credDir, err := p.Config.GetCredentialsDir()
+	require.NoError(t, err, "should have been able to get credentials directory path")
```

Comment:
tiniest of nits on both these tests... perhaps separate logical blocks with empty lines  😬 ?  

But both added tests 💯 !